apps/meteor/ee/client/views/admin/users/SeatsCapUsage/SeatsCapUsage.tsx

```diff
@@ -26,7 +26,7 @@ const SeatsCapUsage = ({ limit, members }: SeatsCapUsageProps): ReactElement =>
 				fontScale='c1'
 				mb='x8'
 			>
-				<div>{t('Seats_Available', { seatsLeft })}</div>
+				<div data-qa-id='seats-available'>{t('Seats_Available', { seatsLeft })}</div>
```

Comment:
Why include this `data-qa-id` if you are not using it? 
Also, check the e2e Wiki for learning more about selector conventions: https://github.com/RocketChat/Rocket.Chat/wiki/E2E-Testing#recommended-selector-usage

Reply:
I'm not using it in this PR, but I'll use that in another PR when I create the E2E tests for the feature. I know the idea is to make the E2E tests in the same PR we are creating the feature, but this is blocked at least until we merge the PR for the federation E2E tests configuration. For the next release for sure, things will happen in the expected flow.

Reply:
Removed data-qa btw.